auth.html

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 </head>
+
 <body>
     <script>
+        // For more information on auth, visit: https://developers.asana.com/docs/authorization
```

Comment:
Somewhat unrelated, but timely: We link to this auth.html example from our developer console and it could use some clarification on what this example is of. Something along the lines of
```
// This is a sample Authentication success HTML response that informs App Components of successful auth.
```

in addition to the comment you already have here.

Reply:
Added!